code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -279,3 +279,27 @@
 		slot_equipment_priority.Insert(index, slot_head)
 		return
 	return ..()
+
+/obj/item/weapon/reagent_containers/glass/beaker/waterbottle

```

Comment:
It uses beaker so it doesn't need to copypaste the code that always the color of the reagant and the filling to change in it